src/Tribe/Event_Tickets/Main.php

```diff
@@ -24,6 +24,13 @@ class Tribe__Events__Event_Tickets__Main {
 	private $attendees_report;
 
 	/**
+	 * Contains an instance of the Ticket Email integration class
+	 * @since 4.0.2
```

Comment:
I do enjoy a bit of space between the description and the @tags but I realize that's slightly subjective :-D